bf: CLDSRV-28 fixed server crash

naren-rajendran opened the pull request:
Cloudserver crashes when a cross origin request is made to put bucket notification configuration with invalid ARN or a configuration with unknown destination.

Commit:
bf: CLDSRV-28 fixed server crash

Cloudserver crashes when a cross origin request is made to put bucket
notification configuration with invalid arn or a configuration with unknown configuration.

bert-e commented:
# Hello naren-rajendran,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/cloudserver/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue CLDSRV-28 contains:


* `7.10.1`



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `7.10.1`

* `8.2.14`


Please check the `Fix Version/s` of CLDSRV-28, or the target
branch of this pull request.

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/CLDSRV-28-put-bucket-notification-crashes-with-cross-origin-requests` into
`development/7.10`
* [w/8.2/bugfix/CLDSRV-28-put-bucket-notification-crashes-with-cross-origin-requests](https://github.com/scality/cloudserver/compare/development/8.2...w/8.2/bugfix/CLDSRV-28-put-bucket-notification-crashes-with-cross-origin-requests) will be merged into `development/8.2` (pull request #3902)



The following branches will **NOT** be impacted:

* `development/6.4`
* `development/7.4`




*Follow* integration pull requests if you would like to be notified of
build statuses by email.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers

Commit:
bf: CLDSRV-28 changed error response

Cloudserver response message to an unknown or unsupported destination in
the request notification configuration is modified to match AWS's
message.

naren-rajendran commented:
/approve

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/7.10`

* :heavy_check_mark: `development/8.2`



The following branches will **NOT** be impacted:

* `development/6.4`
* `development/7.4`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/7.10`

* :heavy_check_mark: `development/8.2`



The following branches have **NOT** changed:

* `development/6.4`
* `development/7.4`



Please check the status of the associated issue CLDSRV-28.

Goodbye naren-rajendran.
